static/main.js

```diff
@@ -194,6 +194,372 @@ const aliases = {
+    "¥": [/(g(et)?_?)?reg(ister)?/],
+    "⇧": [/up(per)?(case)?/,/plus_?2|2_?plus/,/grade_?up/],
+    "Þ⇧": [/(is_?)?strict(ly)?_?(inc(reasing)?|asc(ending)?)/],
+    "⇩": [/low(er)?(case)?/,/minus_?2|2_?minus/,/grade_?down/],
+    "Þ⇩": [/(is_?)?strict(ly)?_?(dec(reasing)?|desc(ending)?)/],
+    "Ǎ": [/(remove|rm)_?non_?alpha/,"2pow"],
+    "ǎ": [/substr(ing)?s?/,/n(th)?_?prime/],
+    "Ǐ": [/dist(inct)?_?prime_?fact(or)?s?/,/append_?(first|head)/],
+    "ǐ": [/prime_?fact(or)?s?/,/sent(ence)?_?case/],
+    "∆ǐ" : [/prime_?exp(onents?)?/],
+    "Ǒ": [/multiplic(ity)?/,/(remove|rm)_?fix(ed)?(_?point)?/,/find_?tru(e|thy)/],
+    "ǒ": [/mod_?(3|three)|(3|three)_?mod/,/chunks?_?2|2_?chunks?/],
+    "Ǔ": [/rot(ate)?_?r(ight)?/],
+    "ÞǓ": [/conn(ected)?_?uniqu(e|ify)?/],
+    "ǔ": [/rot(ate)?_?l(eft)?/],
+    "Þǔ": ["untruth"],
+    "⁽": [/l(am(bda)?)?_?1|1_?l(am(bda)?)?/],
+    "‡": [/l(am(bda)?)?_?2|2_?l(am(bda)?)?/],
+    "≬": [/l(am(bda)?)?_?3|3_?l(am(bda)?)?/],
+    "⁺": [/n(ext)?_?cha?r_?(num(ber)?|int)/],
+    "↵": [/split_?(newline|nl)/,/10_?pow|pow_?10/],
+    "⅛": [/glob(al)?_?push/],
+    "¼": [/glob(al)?_?pop/],
+    "¾": [/push_?glob(al)?/],
+    "Þ¾": [/clear_?glob(al)?/],
+    "Π": [/prod(uct)?/],
+    "„": [/(rot(ate)?_?)?stack_?l(eft)?/],
+    "‟": [/(rot(ate)?_?)?stack_?r(ight)?/],
+    "kF": ["fizzbuzz"],
+    "Þ!": [/(all_?)?fact(orial)?s/],
+    "∆%": [/mod(ular)?_?exp(onent(iation)?)?/],
+    "¨*": [/(all_?)?multi(ple)?s/],
+    "¨,": [/print_?space/],
+    "ø.": ["surround"],
+    "Þ/": [/m(ain)?_?diag(onal)?/],
+    "k1": ["1e3","1000"],
+    "k2": ["1e4","10000"],
+    "k3": ["1e5","100000"],
+    "k4": ["1e6","1000000"],
+    "¨=": [/eq(ual)?_?under/],
+    "Þ<": [/less_inc(reasing)?/],
+    "¨<": [/strict_?(l(ess)?_?t(han)?)/],
+    "¨>": [/strict_?(g(reater)?_?t(han)?)?/],
+    "¨?": [/read_?stdin/],
+    "øA": [/letter_?num(ber)?/,/num(ber)?_?letter/],
+    "ÞA": [/adj(acency)?_?mat(rix)?/],
+    "øB": ["bracketify"],
+    "ÞB": [/rand(om)?_?bits/],
+    "∆C": ["arccos"],
+    "øC": [/num_?compress/],
+    "∆D": [/to_?deg(rees)?/],
+    "øD": [/dict_?comp(ress)?/],
+    "ÞD": [/diag(onal)?s?/],
+    "∆E": [/(e_?power|power_?e)m1/,"expm1"],
+    "øE": [/ends_?with/],
+    "∆F": [/fib(onacci)?_?0/],
+    "ÞF": [/a(ll)?_?fib(onacci)?/],
+    "ÞG": ["longest"],
+    "∆I": [/pi_?digits|digits_?pi/],
+    "ÞI": [/all_?indices_?multi(dimensional)?/],
+    "øJ": ["json"],
+    "∆K": [/sum_?div(isors)?|div(isors)?_?sum/],
+    "ÞK": [/suffix(es)?/],
+    "∆L": [/nat(ural)?_?log|log_?nat(ural)?|log_?e|e_?log/,"ln"],
+    "øL": [/strip_?left/],
+    "∆M": ["mode"],
+    "øM": [/vert(ical)_?flip_?palindrom(is)?e/],
+    "ÞM": [/max(imal)?_?ind(ice)?s?/],
+    "ÞN": [/alt(ernating)?_?neg(ation)?/],
+    "∆P": [/solve_?poly/],
+    "øP": [/plurali[sz]e/],
+    "∆Q": [/solve_?quad(ratic)?/],
+    "∆R": [/to_?rad(ians)?/],
+    "øR": [/strip_?right/],
+    // How does ÞR
+    "∆S": ["arcsin"],
+    "øS": [/strip_?(left|right)/],
```

Comment:
```suggestion
    "øS": [/strip_?both/],
```
it's not stripleft or stripright, it's both

Reply:
Damnit, copilot